src/components/parameters.js

```diff
@@ -125,36 +123,59 @@ const Parameters = ({ showParameters, hideParameters }) => {
 
     const handleChangeTheme = (event) => {
         const theme = event.target.value;
-        dispatch(selectTheme(theme));
+        let json = JSON.stringify([
```

Comment:
Why do you strinify the object here ? This is really ugly in the code and cumbersome. If you need to stringify just to put it in the body (not sure this is necessaey, to check) why not doing it in the  updateConfigParameters function?